.travis.yml

```diff
@@ -1,18 +1,16 @@
 sudo: false
 language: python
+cache: pip
 python:
-  - "2.7"
-  - "3.3"
-  - "3.4"
-  - "3.5"
-  - "3.6"
+  - 2.7
+  - 3.6
 install:
-  - "pip install pytest pytest-cov nose"
-  - "pip install coveralls"
-  - "pip install -e ."
-script: 
-  - python -m pytest affine/tests --cov affine --cov-report term-missing
+  - pip install wheel --upgrade
```

Comment:
I think Travis installs `wheel` by default, otherwise the `bdist_wheel` subcommand is not available.  Any idea if the last release tag was auto-deployed by Travis?

Reply:
@geowurster yes, it was, from the python 3.5 job: https://travis-ci.org/sgillies/affine/jobs/252792681#L333.